topdown/bits_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 The OPA Authors.  All rights reserved.
```

Comment:
[nit]
```suggestion
// Copyright 2020 The OPA Authors.  All rights reserved.
```

I guess? 😉

Reply:
Done!